ui/dashboard.html

```diff
@@ -617,7 +617,7 @@
                 </div>
                 <div class="manage-officers-tags-container">
                     <div style="display: flex; align-items: center;">
-                        <div class="tags-title">Officers Involved</div>
+                        <div class="tags-title" id="bolos-officers-involved-tag-tittle">Officers Involved</div>

```

Comment:
nit typo `title`

Reply:
Idk what u mean by that, but what i did is tried on my local and recorded

Reply:
You misspelled the world `title` as `tittle`. This is a nitpick comment that you should spell it correctly. 
```suggestion
                        <div class="tags-title" id="bolos-officers-involved-tag-title">Officers Involved</div>

```

Reply:
Then you will need to update the other areas using this id